### ISSUE TITLE
State of the project

### ISSUE THREAD

**Issue by bbatsov** [djblue/nrepl-cljs#2]
Hey there!

I was curious what's the state of the project and namely is it in a good enough shape to work with CIDER? I've been working these days on making CIDER a bit more aware of other nREPL servers (starting with Babashka's) and if this one supports the core ops I can add some support for it as well. 

Might be a good idea to mention something about this in the README. 

Also - you can consider implementing some version of `:completions` and `:lookup` (coming in nREPL 0.8) as well.

**Comment by bbatsov** [comment]
Btw, I tried to run the server myself by `npm start` doesn't work. Might be a good idea to update those instructions and perhaps package the server as a package with a CLI at some point.

**Comment by djblue** [comment]
Hi @bbatsov, sorry for the late response. The project is very much a proof of concept for now. I was hoping this project would complement [lumo](https://github.com/anmonteiro/lumo) but interest seems to have died down for that.

I was in the middle of using shadow-cljs to bootstrap this project instead of lumo and forgot to update the npm scripts. I'll update the instructions to reflect that soon. I think I even managed to put together a [nrepl client](https://github.com/djblue/nrepl-cljs/blob/master/src/nrepl/client/tty.cljs) as well.

I'll see if I can find some time to update this project a little.

**Comment by bbatsov** [comment]
> Hi @bbatsov, sorry for the late response. The project is very much a proof of concept for now. I was hoping this project would complement lumo but interest seems to have died down for that.

Yeah, it seems you're right. I didn't follow Lumo closely, but the exit of its author seemed a death sentence for the project given the lack of other significant contributors. 

> I was in the middle of using shadow-cljs to bootstrap this project instead of lumo and forgot to update the npm scripts. I'll update the instructions to reflect that soon. I think I even managed to put together a nrepl client as well.

Hmm, that's an interesting development and given shadow's immense popularity these days, I think it'd be a great move. I'm also sure @thheller would to be thrilled to hear this. Looking forward to seeing how things are going to shape up!

If you create some tickets with outstanding items that need to be tackled I can circulate them around and see if we can't get some people from the broader community to help you with those.

**Comment by djblue** [comment]
The project should be usable again and the readme reflects the shadow-cljs usage. I'll put together some issues soon. Thanks for reaching out and offering help!

**Comment by bbatsov** [comment]
Thanks!

> Currenly, the only real op supported by the server is eval. Contributions are welcome!

Btw, I assume `clone` is supported as well, so clients could actually connect to the server, right?
